src/NServiceBus.NHibernate.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -25,6 +25,23 @@ namespace NServiceBus.Features
         protected override void Setup(NServiceBus.Features.FeatureConfigurationContext context) { }
     }
 }
+namespace NServiceBus
+{
+    public interface INHibernateStorageSession
+    {
+        NHibernate.ISession Session { get; }
+        void OnSaveChanges(System.Func<NServiceBus.Persistence.SynchronizedStorageSession, System.Threading.Tasks.Task> callback);
```

Comment:
Is the `OnSaveChanges` added for completness?

Reply:
It was always there but I changed the name of the interface from `INHibernateSynchronizedStorageSession` to `INHibernateStorageSession` when making it public. Before this PR, the API was available via extension methods on `SynchronizedStorageSessionExtensions`.